packages/dso-toolkit/src/components/highlight-box/highlight-box.mixins.scss

```diff
@@ -12,6 +12,17 @@
   margin-top: units.$u2;
   padding: units.$u2;
   margin-bottom: units.$vertical-spacing-large;
+
```

Comment:
Dus als de custom-properties invalid zijn (niet bestaan in het geval de highlight-box niet onder een dso-equal-heights hangt) dan wordt unset gebruikt?

Reply:
yes. dat is de opzet